app/views/course/announcements/_announcement.html.slim

```diff
@@ -0,0 +1,20 @@
+div class=(announcement.currently_valid? ? '' : 'future')
```

Comment:
OK, but I think we only highlight future announcements ?

expired ones are just don't display on course home page.

On Thu, Jan 22, 2015 at 11:38 AM, Joel Low <notifications@github.com> wrote:

> In app/views/course/announcements/_announcement.html.slim
> <https://github.com/Coursemology/coursemology2/pull/62#discussion_r23353882>
> :
>
> > @@ -0,0 +1,20 @@
> > +div class=(announcement.currently_valid? ? '' : 'future')
>
> might not want to use future, since it will also apply to expired
> announcements.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Coursemology/coursemology2/pull/62/files#r23353882>.
>